packages/desktop/src/renderer/components/Settings/SettingsComponent.tsx

```diff
@@ -145,18 +145,18 @@ export const SettingsComponent: React.FC<SettingsComponentProps> = ({
               <Tab value='notifications' label='Notifications' data-testid={'notifications-settings-tab'} />
               <Tab value='invite' label='Invite a friend' data-testid={'invite-settings-tab'} />
               <Tab value='qrcode' label='QR Code' data-testid={'qr-code-tab'} />
-              {!isWindows && (
-                <Grid style={{ marginTop: '24px', cursor: 'pointer' }}>
-                  <Typography
-                    data-testid='leave-community-tab'
-                    className={classes.leaveComunity}
-                    onClick={leaveCommunityModal.handleOpen}
-                  >
-                    Leave community
-                  </Typography>
-                </Grid>
-              )}
```

Comment:
Apparently, when nesting certain things inside of material components that are not expecting it, weird errors can happen where it passes attributes to the DOM that it shouldn't. I figured this out by stumbling upon an issue that was describing something similar. I'm not sure where that issue went now, but can probably find it. Of course, it's easy to see how this causes errors.